homeassistant/components/growatt_server/__init__.py

```diff
@@ -1 +1,56 @@
 """The Growatt server PV inverter sensor integration."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import CONF_PLANT_ID, DOMAIN
+
+CONFIG_SCHEMA = vol.Schema(
```

Comment:
We shouldn't introduce a new schema just to import it then. Instead initiate a config entry import from `async_setup_platform` from `sensor.py`

Reply:
Oh I see `async_setup_platform` still sets up the integration. In that case, I suggest you remove this schema + all import logic and it will be good to go 👍